scripts/deleteOldVideo.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+VIDEO_DIR="/home/noSleepDrive/videos"
+
+find "$VIDEO_DIR" -maxdepth 1 -type d -regextype posix-extended -regex ".*/[0-9]{4}\.[0-9]{2}\.[0-9]{2}" | while read -r dir
+do
+    folder_date=$(basename "$dir")
+
+    formatted_date=$(echo "$folder_date" | sed 's/\./-/g')
```

Shellcheck (0.10.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)